### ISSUE TITLE
Setting up CKAN with Docker-Compose '\connect: FATAL: database "datastore" does not exist'

### ISSUE THREAD

**Issue by Perla-Al-Haddad** [ckan/ckan#6850]
**CKAN version**
2.9.5

**Describe the bug**

I've been following the documentation to set up CKAN using Docker-Compose on Windows 10
I've reached this part of the installation process: 
> https://docs.ckan.org/en/2.9/maintaining/installing/install-from-docker-compose.html#datastore-and-datapusher

When running the following command:
`docker exec ckan /usr/local/bin/ckan -c /etc/ckan/production.ini datastore set-permissions | docker exec -i db psql -U ckan`

I get the following response: 
> 2022-05-10 09:40:17,758 INFO  [ckan.cli] Using configuration file /etc/ckan/production.ini
> 2022-05-10 09:40:17,759 INFO  [ckan.config.environment] Loading static files from public
> 2022-05-10 09:40:17,829 INFO  [ckan.config.environment] Loading templates from /usr/lib/ckan/venv/src/ckan/ckan/templates
> 2022-05-10 09:40:18,322 INFO  [ckan.config.environment] Loading templates from /usr/lib/ckan/venv/src/ckan/ckan/templates
> \connect: FATAL:  database "datastore" does not exist

**Steps to reproduce**

Following the steps described in the documentation: 
> https://docs.ckan.org/en/2.9/maintaining/installing/install-from-docker-compose.html

**Expected behavior**

It is excepted for the command to run successfully, to be able to enable the datastore in the production.ini file.

**Additional details**
If possible, please provide the full stack trace of the error raised,  or add screenshots to help explain your problem.

I need to setup the datastore to be able to make views for datasets with uploaded or linked data.
For now I am only able to make views for remote CSV files, any other local or remote file type (JSON, XLSX, XLS...) results in the following message in CKAN:

![SS CKAN error](https://user-images.githubusercontent.com/89902296/167604536-b4d5e0e0-2988-4564-8041-fa0b3b05aee9.PNG)

**Comment by kowh-ai** [comment]
This should work. Did you have any problems when building the docker images _or_  running the containers?

**Comment by Perla-Al-Haddad** [comment]
I ran into a couple problems when running the ckan container.

One issue was converting all the files in the solution from CRLF to LF to get rid of the "no such file or directory" error.
After that the ckan container started throwing this error: 

> ImportError: No module named importlib_metadata

I was able to resolve this by following the steps described here #5874
After that the ckan container ran successfully.

The db container is also throwing this error:

> UTC [36] ERROR:  relation "migrate_version" does not exist at character 21
